cli/.js/src/main/scala/aqua/RunCommand.scala

```diff
@@ -38,34 +44,80 @@ object RunCommand extends Logging {
    * @param air code to call
    * @return
    */
-  def funcCall(multiaddr: String, air: Generated, config: TransformConfig)(implicit
+  def funcCall(multiaddr: String, air: String, functionDef: FunctionDef, config: RunConfig)(implicit
     ec: ExecutionContext
-  ): Future[Validated[String, Unit]] = {
+  ): Future[Unit] = {
     (for {
       _ <- Fluence
-        .start(multiaddr)
+        .start(PeerConfig(connectTo = multiaddr))
         .toFuture
       peer = Fluence.getPeer()
+      promise = Promise.apply[Unit]()
       _ = CallJsFunction.registerUnitService(
         peer,
-        "console",
-        "print",
-        args => println("print: " + args)
+        config.consoleServiceId,
+        config.printFunctionName,
```

Comment:
This seems to be not just `console` service, but a completion signal. Maybe call it something like `wrapCallbackSrv` with value like `--after-callback-srv-service--` and function name `print-and-stop`